src/test/java/org/symphonyoss/symphony/messageml/elements/MentionTest.java

```diff
@@ -177,12 +177,16 @@ public void testSoftMentionInvalidEmail() throws Exception {
     String invalidAttr = "<messageML>Hello <mention email=\"invalid@email.com\" strict=\"false\" />!</messageML>";
     context.parseMessageML(invalidAttr, null, MessageML.MESSAGEML_VERSION);
 
+    String expectedJson = "{\"urls\":[{\"text\":\"mailto:invalid@email.com\","
```

Comment:
Please refer to https://github.com/symphonyoss/messageml-utils/pull/66/files for additional unit tests.